docs/walkthrough.md

```diff
@@ -142,24 +129,7 @@ However, the key `script` in should contain the actual bytecode to execute, i.e.
 python3 -c 'print(list(open("out/debug/dbg_example.bin", "rb").read()))'
```

Comment:
On line 126, I think it should be "However, the key is `script` should contain"

Reply:
Good catch